### ISSUE TITLE
devopsdays Amsterdam CFP: 30 April

### ISSUE THREAD

**Issue by ksatirli** [ansible/community#35]
The fourth edition of devopsdays Amsterdam: http://www.devopsdays.org/events/2016-amsterdam/propose/

CFP closes April 30th.

**Comment by ksatirli** [comment]
(I'd have this assigned to you, @robynbergeron, but I do not have the rights!)

**Comment by robynbergeron** [comment]
Hi @ksatirli -- thanks for submitting this one. 

I guess my question to you is: *should* it be assigned to me, and why? :smiley: 

A few thoughts: 

* Greg and I have been opening issues for CFPs for things that either he or I have been intending to submit for and assigning them to ourselves. 
* But: We've also talked in the past about what else we can do with CFPs -- should we have a once-a-month newsletter, or blog post, where we can tell folks in the Ansible community to submit proposals for CFPs? 

The thing is -- particularly with DevOpsDays events, the best talks (and the most likely talks to get accepted) are not about *tools themselves*, but more often about how people are doing magical things in their organizations, and often talk about the tools they use to do those magical things. Hopefully Ansible. But the point is that... I am not doing magical things with Ansible, per se, certainly not as magical as the things that you and many others are doing :)

@ksatirli -- what do you think the best thing for us to do here would be? We could:
* Leave it open and say "anyone who wants to submit, please do so and let us know here if you did and if it gets accepted" ... maybe assign it to them? But ideally we don't want to imply that "only one person should submit."
* Assign it to someone who wants to try to get others to submit a talk for DevOpsDays? (And open up issues for anyone else plans wants to submit just so we can help remind them?)
* Add this to some sort of monthly newsletter or blog post? (Which isn't being written right now, but hey, we could open up an issue for that and find someone who might want to do that?)
* Some other option?
* Any combination of the above?

**Comment by robynbergeron** [comment]
Also -- @ksatirli -- my thought right now with issues is that they can probably sit unassigned until someone volunteers to take it on. It could be me, or greg, but our goal in having this group is generally to let other people also be able to take tasks on, unless they are things we are explicitly assigning to ourselves after we create them. There's plenty to do and lots that we haven't even imagined yet :)

If that's not clear, maybe we can fix up the readme.md or contributors.md file somehow.

**Comment by ksatirli** [comment]
Hey @robynbergeron, thank you so much for the extensive answer! 

First: should it be assigned to you? I am unsure! My goal was to attempt to duplicated what you did in #34 to allow you and @gregdek to triage this in a better way.

[](url)

**Comment by robynbergeron** [comment]
Okay. Well -- I think (sadly) neither Greg nor I submitted for this -- though I'm sure others did, so I am closing this. 

But: I did open a new issue for us to figure out a better way of effectively sharing CFPs with the wider ansible community so that we can all submit and benefit and so forth. 

https://github.com/ansible/community/issues/89 is the new thing. :)

Thanks @ksatirli for submitting this -- hopefully some Ansible-y things came this way regardless, I'm pretty sure it's quite possible :D and hopefully we can figure out how to more broadly advertise things soon.
